hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -23,6 +23,87 @@ To expand an issue number into a full GitHub link, use this template:
 
 https://github.com/hazelcast/hazelcast-jet/issues/<issue number>
 
+
+ADD YOUR PR ENTRY IN THIS SECTION: >>>
```

Comment:
I like!

Reply:
The instructions are a part of the New PR template that I just updated.